moit-api/src/main/kotlin/com/mashup/moit/facade/StudyFacade.kt

```diff
@@ -69,4 +71,16 @@ class StudyFacade(
     fun checkFirstAttendance(studyId: Long): StudyFirstAttendanceResponse {
         return StudyFirstAttendanceResponse.of(attendanceService.existFirstAttendanceByStudyId(studyId))
     }
+
+    @Transactional
+    fun adjustAbsenceStatus(studyIds: Set<Long>) {
+        studyService.findByStudyIds(studyIds.toList())
+            .groupBy { study -> study.moitId }
+            .forEach { (moitId, studies) ->
+                studies.forEach { study ->
+                    attendanceService.adjustUndecidedAttendancesByStudyId(study.id)
+                        .forEach { attendanceId -> fineService.create(attendanceId, moitId) }
+                }
+            }
```

Comment:
```suggestion
        studyService.findByStudyIds(studyIds.toList()).forEach {
            attendanceService.adjustUndecidedAttendancesByStudyId(it.id)
                .forEach { attendanceId -> fineService.create(attendanceId, it.moitId) } 
        }
```

이 로직이랑 동일한 로직인거 같은데 좀 더 간결하게 줄이면 어떨까요~?